src/Forms/__tests__/ValidationErrorMessage.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, cleanup } from 'test-utils';
+import ValidationErrorMessage from '../ValidationErrorMessage';
+
+describe('Validation Error Message', function () {
+  let getByText;
```

Comment:
This should be typed

Reply:
Added type through commit 10369ab! Thanks